src/containers/teamPage/index.tsx

```diff
@@ -0,0 +1,268 @@
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 80px;
+`;
+
+const PageHeaderContainer = styled.div`
+  max-width: 600px;
+`;
+
+const JoinButton = styled(Button)`
+  width: 125px;
+  height: 45px;
+  color: ${WHITE};
+  background-color: ${MID_GREEN};
+  font-size: 18px;
+`;
+
+const CenterDiv = styled.div`
+  margin-top: -50px;
+  display: flex;
+  justify-content: space-between;
+`;
+
+const MemberContainer = styled.div`
+  width: 45%;
+`;
+
+const GoalContainer = styled.div`
+  width: 55%;
+`;
+
+const SectionHeader = styled(Paragraph)`
+  font-size: 25px;
+  font-weight: bold;
+  line-height: 1;
+  color: ${DARK_GREEN};
+`;
+
+const StyledMemberList = styled(List)`
+  border-bottom: 0px;
+`;
+
+const LeaderboardCollapse = styled(Collapse)`
+  color: ${BLACK};
+  font-size: 15px;
+`;
+
+const BlackText = styled(Paragraph)`
+  display: inline-block;
+  color: ${BLACK};
+  font-size: 15px;
+  line-height: 0.5;
+`;
+
+const StyledPanel = styled(Panel)`
+  min-height: 40px;
+  margin-bottom: 10px;
+  padding-left: 10px;
+  background-color: ${LIGHT_GREEN}90;
+  border: 0px;
+`;
+
+const Line = styled.div`
+  height: 1px;
+  width: 104%;
+  margin: 0px -10px 10px -20px;
+  display: block;
+  background-color: ${WHITE};
+`;
+
+// dummy data
+const sampleMemberData: MemberProps[] = [
+  {
+    user_id: 1,
+    username: 'florisdobber',
+    team_role: TeamRole.LEADER,
+  },
+  {
+    user_id: 2,
+    username: 'jackblanc',
+    team_role: TeamRole.MEMBER,
+  },
+  {
+    user_id: 3,
+    username: 'atreecounter',
+    team_role: TeamRole.NONE,
+  },
+  {
+    user_id: 4,
+    username: 'speakerofthetrees',
+    team_role: TeamRole.PENDING,
+  },
+  {
+    user_id: 5,
+    username: 'vrushalitarte01',
+    team_role: TeamRole.MEMBER,
+  },
+  {
+    user_id: 6,
+    username: 'willmt80',
+    team_role: TeamRole.MEMBER,
+  },
+  {
+    user_id: 7,
+    username: 'apradhan12',
+    team_role: TeamRole.MEMBER,
+  },
+  {
+    user_id: 8,
+    username: 'SofieDunt',
+    team_role: TeamRole.MEMBER,
+  },
+];
+
+const sampleGoalData: GoalProps[] = [
+  {
+    id: 1,
+    goal: 50,
+    progress: 5,
+    start_date: new Date(2021, 0, 1),
+    complete_by: new Date(2022, 0, 1),
+    completion_date: null,
+  },
+  {
+    id: 2,
+    goal: 500,
+    progress: 71,
+    start_date: new Date(2021, 2, 20),
+    complete_by: new Date(2021, 4, 21),
+    completion_date: null,
+  },
+  {
+    id: 3,
+    goal: 10,
+    progress: 10,
+    start_date: new Date(2020, 5, 10),
+    complete_by: new Date(2020, 7, 10),
+    completion_date: new Date(2020, 6, 13),
+  },
+];
+
+const sampleTeamData: TeamProps = {
+  id: 1,
+  name: 'Code4Community',
+  bio: 'Short description about how cool of a team Code4Community is!',
+  members: sampleMemberData,
+  goals: sampleGoalData,
+};
+
+const TeamPage: React.FC = () => {
+  const onClick = () => {
+    // TODO: join team
+  };
+
+  return (
+    <TeamContainer>
+      <StyledLinkButton to={Routes.LOGIN}>
```

Comment:
📌  This should be a different route, no? To the "All Teams" page? Can you drop a ticket to track the work to change this?